### PR TITLE
Align v5 docs on develop with v5 branch

### PR DESCRIPTION
**Pull request by bychkovdmitry**
### 🎯 Goal

Reset some pages to the state they were in v5.

#### General
- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] Assigned a person / code owner group (required)
- [x] Thread with the PR link started in a respective Slack channel (#android-chat-core or #android-chat-ui) (required)
- [x] PR targets the `develop` branch
- [ ] PR is linked to the GitHub issue it resolves

#### Code & documentation
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [x] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] UI Changes correct (before & after images)
- [ ] Bugs validated (bugfixes)
- [ ] New feature tested and works
- [ ] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

### CODE REVIEW

File: docusaurus/android_versioned_docs/version-5.x.x/05-resources/01-sample-apps.mdx [hunk 1]
```diff
@@ -1,45 +0,0 @@
-# Sample Apps
```
Comment (bychkovdmitry) [inline docusaurus/android_versioned_docs/version-5.x.x/05-resources/01-sample-apps.mdx:1]: Let's have this in v5

File: docusaurus/android_versioned_docs/version-5.x.x/04-compose/07-guides/06-implementing-own-capabilities.mdx [hunk 2]
```diff
@@ -64,7 +64,7 @@ found [here](https://github.com/GetStream/stream-chat-android/blob/0f1f54e912baf
 :::note
 Own capabilities are publicly exposed inside `MessageComposerController` and `MessageComposerViewModel` as `ownCapabilities: StateFlow<Set<String>>`.
 
-For this component, as well as all the other examples, the capabilities in bold are a part of the "default" behavior that is set up for most users in the dashboard.
```
Comment (bychkovdmitry) [inline docusaurus/android_versioned_docs/version-5.x.x/04-compose/07-guides/06-implementing-own-capabilities.mdx:67]: Let's have this in v5

File: docusaurus/android_versioned_docs/version-5.x.x/01-basics/02-choosing-the-correct-sdk.mdx [hunk 3]
```diff
@@ -231,9 +231,9 @@ Performance wise, none of these results exclude the other and both SDKs show no
 It should be noted that the View toolkit has been active for more than a decade and has been the native way of rendering UI in Android, so it's bound to have better performance
 in some scenarios.
 
-However, Jetpack Compose is a much more recent technology and as such, it will keep
-receiving performance improvements in the future. It already outperforms our UI Components SDK in certain aspects and if you choose our Compose SDK, you should expect a performance
-bump with every significant improvement of the Jetpack Compose toolkit.
+However, Jetpack Compose has been out for less than a year - it's a much more recent technology and that as such, it will keep
+receiving performance improvements in the future. It already outperforms our UI Components SDK in some aspects, but if you choose our Compose SDK, you should expect a performance
```
Comment (MarinTolic) [inline docusaurus/android_versioned_docs/version-5.x.x/01-basics/02-choosing-the-correct-sdk.mdx:235]: Keep the updated version.

File: docusaurus/android_versioned_docs/version-5.x.x/04-compose/07-guides/01-adding-custom-attachments.mdx [hunk 4]
```diff
@@ -13,7 +13,7 @@ This involves doing the following steps:
 :::note
 In this guide, we'll show only the main points concerning custom attachments and their factories. Smaller parts will be omitted for the sake of being concise.
 
-You can find the full code from this guide on [GitHub](https://github.com/GetStream/stream-chat-android/tree/main/stream-chat-android-ui-guides/src/main/java/io/getstream/chat/android/guides/catalog/compose/customattachments). To check the final result, clone the repository and run the `stream-chat-android-ui-guides` module.
```
Comment (bychkovdmitry) [inline docusaurus/android_versioned_docs/version-5.x.x/04-compose/07-guides/01-adding-custom-attachments.mdx:16]: Let's have this in v5 as Android-Samples repo contains no guides anymore

File: docusaurus/android_versioned_docs/version-5.x.x/04-compose/05-message-components/03-message-list.mdx [hunk 5]
```diff
@@ -176,12 +176,11 @@ fun MessageList(
     itemContent: @Composable (MessageListItemState) -> Unit = { messageListItem ->
         DefaultMessageContainer(
             messageListItem = messageListItem,
-            onMediaGalleryPreviewResult = onMediaGalleryPreviewResult,
+            onImagePreviewResult = onImagePreviewResult,
```
Comment (bychkovdmitry) [inline docusaurus/android_versioned_docs/version-5.x.x/04-compose/05-message-components/03-message-list.mdx:179]: Should be reverted in v5

File: docusaurus/android_versioned_docs/version-5.x.x/04-compose/05-message-components/03-message-list.mdx [hunk 6]
```diff
@@ -116,8 +116,8 @@ MessageList(
     onLastVisibleMessageChanged = { message -> },
     onScrollToBottom = { },
     onGiphyActionClick = { giphyAction -> },
-    onMediaGalleryPreviewResult = { mediaGalleryPreviewResult -> },
-     // Content
+    onImagePreviewResult = { imagePreviewResult -> },
```
Comment (bychkovdmitry) [inline docusaurus/android_versioned_docs/version-5.x.x/04-compose/05-message-components/03-message-list.mdx:119]: Should be reverted in v5

File: docusaurus/android_versioned_docs/version-5.x.x/04-compose/05-message-components/01-messages-screen.mdx [hunk 7]
```diff
@@ -23,12 +23,7 @@ override fun onCreate(savedInstanceState: Bundle?) {
 
     setContent {
         ChatTheme {
-            MessagesScreen(
-                viewModelFactory = MessagesViewModelFactory(
-                    context = this,
-                    channelId = channelId
-                )
-            )
+            MessagesScreen(channelId = channelId)
```
Comment (bychkovdmitry) [inline docusaurus/android_versioned_docs/version-5.x.x/04-compose/05-message-components/01-messages-screen.mdx:26]: Should be reverted in v5

File: docusaurus/android_versioned_docs/version-5.x.x/01-basics/04-getting-started.mdx [hunk 8]
```diff
@@ -6,14 +6,14 @@ import TabItem from '@theme/TabItem';
 Let's see how you can get started with the Android Chat SDK after adding the required [dependencies](./03-dependencies.mdx). This page shows you how to initialize the SDK in your app.
```
Comment (MarinTolic) [inline docusaurus/android_versioned_docs/version-5.x.x/01-basics/04-getting-started.mdx:7]: keep the updated version

File: docusaurus/android_versioned_docs/version-5.x.x/03-ui/06-guides/10-customizing-message-list-scroll-behavior.mdx [hunk 9]
```diff
@@ -1,67 +0,0 @@
-# Customizing MessageListView Scrolling Behavior
```
Comment (bychkovdmitry) [inline docusaurus/android_versioned_docs/version-5.x.x/03-ui/06-guides/10-customizing-message-list-scroll-behavior.mdx:1]: Let's have this in v5

File: docusaurus/android_versioned_docs/version-5.x.x/03-ui/04-message-components/02-message-list.mdx [hunk 10]
```diff
@@ -3,43 +3,47 @@ import TabItem from '@theme/TabItem';
 
 # Message List
```
Comment (MarinTolic) [inline docusaurus/android_versioned_docs/version-5.x.x/03-ui/04-message-components/02-message-list.mdx:4]: keep updated

File: docusaurus/android_versioned_docs/version-5.x.x/04-compose/05-message-components/01-messages-screen.mdx [hunk 11]
```diff
@@ -13,7 +13,7 @@ The `MessagesScreen` component is the second screen component in the SDK, out of
 
 ## Usage
```
Comment (MarinTolic) [inline docusaurus/android_versioned_docs/version-5.x.x/04-compose/05-message-components/01-messages-screen.mdx:15]: use v5 version

File: docusaurus/android_versioned_docs/version-5.x.x/04-compose/05-message-components/03-message-list.mdx [hunk 12]
```diff
@@ -85,8 +85,8 @@ fun MessageList(
     onLastVisibleMessageChanged: (Message) -> Unit = { viewModel.updateLastSeenMessage(it) },
```
Comment (MarinTolic) [inline docusaurus/android_versioned_docs/version-5.x.x/04-compose/05-message-components/03-message-list.mdx:85]: use v5 version

File: docusaurus/android_versioned_docs/version-5.x.x/03-ui/03-channel-components/02-channel-list.mdx [hunk 13]
```diff
@@ -221,8 +221,8 @@ The transformer should be set before the View is rendered to make sure that the
     android:layout_height="64dp"
```
Comment (bychkovdmitry) [inline docusaurus/android_versioned_docs/version-5.x.x/03-ui/03-channel-components/02-channel-list.mdx:221]: Keep v5 version

File: docusaurus/android_versioned_docs/version-5.x.x/03-ui/06-guides/05-app-startup-initializers.mdx [hunk 14]
```diff
@@ -21,6 +21,10 @@ Specifically, you'll have to do the following:
         android:authorities="${applicationId}.androidx-startup"
         android:exported="false"
         tools:node="merge">
+        <meta-data
```
Comment (bychkovdmitry) [inline docusaurus/android_versioned_docs/version-5.x.x/03-ui/06-guides/05-app-startup-initializers.mdx:24]: Keep v5 version

**Review by MarinTolic — APPROVED** [review]

**Pull request merged**